CRM/Mailing/BAO/MailingJob.php

```diff
@@ -478,6 +478,40 @@ public function queue($testParams = NULL) {
   }
 
   /**
+   * Build a join and where part for a query
+   *
+   * @param $contact_id
+   * @return array - the first key is join part of the query and the second key is the where part of the query
+   */
+  public function buildAcl($contact_id) {
```

Comment:
This function looks a lot like _buildAcl_ method added above. Can't we re-use the same method instead of creating a new?